spec/models/artist_detail_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe ArtistDetail, type: :model do
+  describe "validations" do
```

Comment:
```suggestion
  describe 'validations' do
```

Let's use single quotes everywhere. Thanks!

Reply:
Updated